easybuild/easyconfigs/n/ncompress/ncompress-4.2.4.4-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,40 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ncompress'
+version = '4.2.4.4'
+
+homepage = 'http://ncompress.sourceforge.net/'
+
+description = """
+ Compress is a fast, simple LZW file compressor. Compress does not have the
+ highest compression rate, but it is one of the fastest programs to compress
+ data. Compress is the defacto standard in the UNIX community for compressing
+ files.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['https://github.com/vapier/ncompress/archive/']
+sources = ['%(name)s-%(version)s.tar.gz']
```

Comment:
Yes, it does a sort of rename, maybe if you use the `releases` URL rather than `archive`?

We actually *have* a template for GitHub URLs, i.e. `GITHUB_SOURCE` see https://github.com/easybuilders/easybuild-framework/pull/1872.

It hasn't been used much though, and it has the caveat of also needing to define `github_account = '...'` in the easyconfig file...